src/api/hooks/useGetMainnetValidators.ts

```diff
@@ -0,0 +1,76 @@
+import {useGlobalState} from "../../GlobalState";
+import {useEffect, useState} from "react";
+import {useGetValidatorSet} from "./useGetValidatorSet";
+
+const MAINNET_VALIDATORS_DATA_URL =
+  "https://aptos-analytics-data-mainnet.s3.amazonaws.com/validator_stats_v1.json";
+
+export interface MainnetValidatorData {
+  owner_address: string;
+  operator_address: string;
+  voting_power: string;
+  governance_voting_record: string;
+  last_epoch: number;
+  last_epoch_performance: string;
+  liveness: number;
+  rewards_growth: number;
+  location_stats: GeoData;
+}
+
+export interface GeoData {
+  peer_id: string;
+  latitude: number;
+  longitude: number;
+  city: string;
+  country: string;
+  region: string;
+  epoch: number;
+}
+
+function useGetMainnetValidatorsRawData() {
+  const [state, _] = useGlobalState();
+  const [validatorsRawData, setValidatorsRawData] = useState<
+    MainnetValidatorData[]
+  >([]);
+
+  useEffect(() => {
+    if (state.network_name === "mainnet") {
+      const fetchData = async () => {
+        const response = await fetch(MAINNET_VALIDATORS_DATA_URL);
+        const data = await response.json();
+        setValidatorsRawData(data);
+      };
+
+      fetchData().catch((error) => {
+        console.error("ERROR!", error, typeof error);
+      });
+    } else {
+      setValidatorsRawData([]);
+    }
+  }, [state]);
+
+  return {validatorsRawData};
+}
+
+export function useGetMainnetValidators() {
+  const {activeValidators} = useGetValidatorSet();
+  const {validatorsRawData} = useGetMainnetValidatorsRawData();
+  const [validators, setValidators] = useState<MainnetValidatorData[]>([]);
+
+  useEffect(() => {
+    if (activeValidators.length > 0 && validatorsRawData.length > 0) {
```

Comment:
do we wanna check whether both `activeValidators` and `validatorsRawData` have equal length? we had sev about this previously cuz we added new validator to the list so that we couldn't find the match for that newly added validator?